themes/bootprint3/theme.config.php

```diff
@@ -1,4 +1,158 @@
+            'view-list' => 'Fugue:view_list.png',
+            'warning' => 'Fugue:error.png',
+            // Formats (Similar items)
+            'format-atlas' => 'Fugue:map.png',
+            'format-book' => 'Fugue:book.png',
+            'format-braille' => 'Fugue:book.png',
+            'format-cdrom' => 'Fugue:cd.png',
+            'format-chart' => 'Fugue:chart_bar.png',
+            'format-chipcartridge' => 'Fugue:computer.png',
+            'format-collage' => 'Fugue:photos.png',
+            'format-default' => 'Fugue:book.png',
+            'format-disccartridge' => 'Fugue:computer.png',
+            'format-drawing' => 'Fugue:edit.png',
+            'format-ebook' => 'Fugue:book_link.png',
+            'format-electronic' => 'Fugue:page_white_zip.png',
+            'format-filmstrip' => 'Fugue:film.png',
+            'format-file' => 'Fugue:page.png',
+            'format-flashcard' => 'Fugue:lightning.png',
+            'format-floppydisk' => 'Fugue:disk.png',
+            'format-folder' => 'Fugue:folder.png',
+            'format-globe' => 'Fugue:world.png',
+            'format-journal' => 'Fugue:page.png',
+            'format-kit' => 'Fugue:box.png',
+            'format-manuscript' => 'Fugue:page_white_text.png',
+            'format-map' => 'Fugue:map.png',
+            'format-microfilm' => 'Fugue:film.png',
+            'format-motionpicture' => 'Fugue:page_white_camera.png',
+            'format-musicalscore' => 'Fugue:music.png',
+            'format-musicrecording' => 'Fugue:music.png',
+            'format-newspaper' => 'Fugue:newspaper.png',
+            'format-online' => 'Fugue:book_link.png',
+            'format-painting' => 'Fugue:page_white_paint.png',
+            'format-photo' => 'Fugue:photo.png',
+            'format-photonegative' => 'Fugue:photos.png',
+            'format-physicalobject' => 'Fugue:bricks.png',
+            'format-print' => 'Fugue:photo.png',
+            'format-sensorimage' => 'Fugue:camera.png',
+            'format-serial' => 'Fugue:layers.png',
+            'format-slide' => 'Fugue:shape_move_forwards.png',
+            'format-software' => 'Fugue:page_white_cd.png',
+            'format-soundcassette' => 'Fugue:sound.png',
+            'format-sounddisc' => 'Fugue:cd.png',
+            'format-soundrecording' => 'Fugue:sound.png',
+            'format-tapecartridge' => 'Fugue:sound.png',
+            'format-tapecassette' => 'Fugue:music.png',
+            'format-tapereel' => 'Fugue:film.png',
+            'format-transparency' => 'Fugue:shape_move_forwards.png',
+            'format-unknown' => 'Fugue:page_red.png',
+            'format-video' => 'Fugue:film.png',
+            'format-videocartridge' => 'Fugue:film.png',
+            'format-videocassette' => 'Fugue:film.png',
+            'format-videodisc' => 'Fugue:cd.png',
+            'format-videoreel' => 'Fugue:film.png',
+            // Currencies
+            'eur' => 'Fugue:money_euro.png',
```

Comment:
I think currencies need a prefix like formats have. Otherwise we have a real chance of confusion (what's 'ils'?).

Reply:
It's the Israeli shekel, I believe. A prefix is in order!